src/widgets/chart.rs

```diff
@@ -90,7 +90,7 @@ pub enum GraphType {
 #[derive(Debug, Default, Clone, PartialEq)]
 pub struct Dataset<'a> {
     /// Name of the dataset (used in the legend if shown)
-    name: Cow<'a, str>,
+    name: Option<Cow<'a, str>>,
```

Comment:
Observation for future - this could be changed to `Line` to support styling the items.